src/types.js

```diff
@@ -31,7 +31,8 @@ export type TXOBJ = {
     value: number,
     address: string,
     n: number,
-    mcReturnAddress: string
+    mcReturnAddress: string,
+    mcReturnPubKey?: string
```

Comment:
We just have one of this. The data is a mainchain pub key hash but also in the zend this field is called mcReturnAddress

Reply:
I changed it in the deserialization too